app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/EventDetailFragment.java

```diff
@@ -120,6 +122,7 @@ public View onCreateView(LayoutInflater inflater,
     public void setArguments(Bundle args) {
         super.setArguments(args);
         day = args.getInt(BundleKeys.EVENT_DAY, 0);
+        eventDuration = args.getString(BundleKeys.EVENT_DURATION);
```

Comment:
Duration should be an `int` instead of `String`. Please use `eventDuration = args.getInt(BundleKeys.EVENT_DURATION, 0);`